build-system/tasks/e2e/package.json

```diff
@@ -6,7 +6,7 @@
   "devDependencies": {
     "@babel/register": "7.15.3",
     "babel-regenerator-runtime": "6.5.0",
-    "chromedriver": "95.0.0",
+    "chromedriver": "98.0.0",
```

Comment:
This allows the e2e tests to work with the latest version of chrome.

Reply:
We'll want to revert this before merging. Circle CI has chrome pinned at 95 for now.

Reply:
I'll leave it for now, or else I have to find another workaround for e2e tests locally.

Reply:
Don't forget to remove this if the PR is ready to go.